spec/features/staff/users_can_edit_an_activity_spec.rb

```diff
@@ -6,6 +6,67 @@
   context "when the activity is a fund" do
     let(:user) { create(:beis_user) }
 
+    context "when the activity is a fund" do
+      it "does not show the parent field" do
+        activity = create(:fund_activity, organisation: user.organisation)
+
+        visit organisation_activity_details_path(activity.organisation, activity)
+
+        expect(page).not_to have_content(I18n.t("summary.activity.parent"))
+      end
+    end
+
+    context "when the activity was left at the level step" do
+      it "shows add link on the level step" do
+        activity = create(:fund_activity, :level_form_state, organisation: user.organisation)
+
+        visit organisation_activity_details_path(activity.organisation, activity)
+
+        within(".level") do
+          expect(page).to have_content(I18n.t("default.link.add"))
+        end
+
+        within(".parent") do
+          expect(page).not_to have_content(I18n.t("default.link.add"))
+        end
+      end
+    end
+
+    context "when the activity was left at the parent step" do
+      it "shows add link on the level step" do
+        activity = create(:programme_activity, :parent_form_state, organisation: user.organisation)
+
+        visit organisation_activity_details_path(activity.organisation, activity)
+
+        within(".parent") do
+          expect(page).to have_content(I18n.t("default.link.add"))
+        end
+
+        within(".identifier") do
+          expect(page).not_to have_content(I18n.t("default.link.add"))
+        end
+      end
+    end
+
+    # Changing the level means the IATI identifier we construct changes
+    # If this is changed after publishing to IATI the effect will be a branch
+    # new IATI publiciation.
+    context "when the activity has a level and a parent" do
```

Comment:
Have we thought about how a user will rectify the situation where they create an activity at the wrong level?

Reply:
We haven't built any of this of course but I was imaginging it would be a similar journey for deleting a record. The user (for now) should contact support and it will end up with the dev team as a request to change the level or remove the record. So long as it remained an edge case it would be the cheapest way to review the implications of deleting/changing hierarchy.